Mathlib/Analysis/Meromorphic/NormalFormAt.lean

```diff
@@ -330,8 +331,11 @@ theorem meromorphicNFAt_toMeromorphicNFAt :
 
 /-- If `f` has normal form at `x`, then `f` equals `f.toNF`. -/
 @[simp] theorem toMeromorphicNFAt_eq_self :
-    MeromorphicNFAt f x ↔ f = toMeromorphicNFAt f x where
+    f = toMeromorphicNFAt f x ↔ MeromorphicNFAt f x where
```

Comment:
... and you seem to have accidentally changed the existing lemma too
```suggestion
    toMeromorphicNFAt f x = f ↔ MeromorphicNFAt f x where
```

Reply:
I needed to change that, in order to avoid a linter warning. In the original form, `simp` would "simplify" `MeromorphicNFAt f x` to ` f = toMeromorphicNFAt f x`, which is probably not what anybody wants.

Reply:
Are you sure? This simp warning sounds rather like you did
```suggestion
    MeromorphicNFAt f x ↔ toMeromorphicNFAt f x = f where
```
or
```suggestion
    MeromorphicNFAt f x ↔ f = toMeromorphicNFAt f x where
```
This is **not** what I am suggesting above